src/Modules/LearningTasks/Tutor.LearningTasks.Core/Domain/Step.cs

```diff
@@ -0,0 +1,23 @@
+﻿using Tutor.BuildingBlocks.Core.Domain;
+
+namespace Tutor.LearningTasks.Core.Domain;
+
+public class Step : Entity
```

Comment:
Value object?

Reply:
This is an Entity because of the StepProgress object, which will reference Step.